Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM debian:stable
+SHELL ["/bin/bash", "-c"]
+
+RUN apt-get update
+RUN apt-get install -y curl git
+
+ENV GOPATH $HOME/work
+ENV PATH $PATH:/usr/local/go/bin:$GOPATH/bin
+
+RUN curl -O https://dl.google.com/go/go1.17.7.linux-amd64.tar.gz
```

Comment:
Could you just use `apt-get` again to get Go instead of getting it this way?  Maybe it saves you from running all of these other steps to  setup Go for this Docker image.  I found the following two for how to get Go using `apt-get` and `apt`.  The examples are meant for Ubuntu, but Ubuntu is based off of Debian so I guess it should just work as well.

https://gist.github.com/Adron/4e1e7d0f71da7c415f455d5930ea94c9
https://github.com/golang/go/wiki/Ubuntu

I guess the only thing is that through the package manager it defaults to getting the latest version of Go at 1.18 and it looks like you are specifically getting version 1.17.7 so if you want to keep it at that version then I guess you can stick to this method or have to set the package manager command to get the 1.17.7 version.

Reply:
Yeah, I didn't even think of that, but I think its good to have a specific version coming in. It doesn't have to be 1.17.1 specifically, but just having the same one each time, and only upgrading intentionally.